.gitignore

```diff
@@ -1,5 +1,5 @@
 *.sw[a-z]
 bazel-*
-integ-tests/integ-tests.iml
-integ-tests/.sw[a-z]
-.ijwb
+integ-tests/teradata/.sw[a-z]
```

Comment:
Line 1 should already match anything that is matched by this.

"If there is a separator at the beginning or middle (or both) of the pattern, then the pattern is relative to the directory level of the particular .gitignore file itself. Otherwise the pattern may also match at any level below the .gitignore level."

https://git-scm.com/docs/gitignore

Reply:
done.